test/EndToEnd/tests/NetCoreProjectTest.ps1

```diff
@@ -230,4 +230,35 @@ function Test-NetCoreToolsVSandMSBuildNoOp {
 
     #Assert
     Assert-True ($MsBuildRestoreTimestamp -eq $VSRestoreTimestamp)
+}
+
+function Test-NetCorePackageVersionNoInclusiveLowerBoundNU1604 {
+    # Arrange
+    $project = New-NetCoreConsoleApp ConsoleApp
```

Comment:
The code you changed is not .NET Core, it's legacy PR.

Reply:
@nkolev92 
Sorry, corrected the problem.